src/models/bases/base-model.ts

```diff
@@ -30,6 +31,10 @@ export abstract class BaseModel extends BaseEntity {
 
     if (this.errors.length <= 0) {
       await super.save(options)
+    } else {
+      this.errors.forEach((e: ValidationError) => {
```

Comment:
I think doing this in production will add too much unnecessary noise to log files?

Reply:
I don't know - we should basically never have database validation errors at this point in Winnie's life, so I'm inclined to leave it in, at least until we get started with challenges?

Reply:
You're going to be the one who has to dig through those logs if / when things break so ultimately, up to you, but validation errors are most likely to be fired by bad user input and that seems like something that can produce a lot of noise

Reply:
Ideally, I want to catch all of those errors in code rather than letting them go to the database, so I say we leave it in until it causes problems.